R/observations.R

```diff
@@ -241,6 +241,84 @@ prepare_data.copula <- function(x, x_test_gaussian = 1, seed = 1, n_samples = 1e
   return(dt)
 }
 
+#' @param n_samples Integer. The number of obs to sample from the leaf if \code{sample} = TRUE or if \code{sample}
+#' = FALSE but \code{n_samples} is less than the number of obs in the leaf.
+#'
+#' @param index_features List. Default is NULL but if either various methods are being used or various mincriterion are
+#' used for different numbers of conditoned features, this will be a list with the features to pass.
+#'
+#' @param  mc_cores Integer. Only for class \code{ctree} currently. The number of cores to use in paralellization of the
+#' tree building and tree prediction. Defaults to 1. Uses parallel::mclapply which relies on forking, i.e. does not
+#' work on Windows systems.
```

Comment:
Add something like: If working on a Windows system, mclapply() defaults to lapply()

Reply:
Suggesting: @param  mc_cores Integer. Only for class \code{ctree} currently. The number of cores to use in paralellization of the tree building (\code{create_ctree}) and tree sampling (\code{sample_ctree}). Defaults to 1. Note: Uses parallel::mclapply which relies on forking, i.e. uses only 1 core on Windows systems.

OK @aredelmeier ? I think it is more important to pinpoint how many cores that are used in windows than what functions it falls back to. On linux mc_cores=1 will also use lapply instead of mclapply.

Reply:
Great.